lib/msal-common/package.json

```diff
@@ -54,7 +54,8 @@
     "build:modules:watch": "rollup -cw",
     "build": "npm run clean && npm run build:modules",
     "build:all": "npm run build",
-    "prepack": "npm run build"
+    "prepack": "npm run updateVersion && npm run build",
+    "updateVersion": "node ../../release-scripts/updateVersion.js"
```

Comment:
Would it be better to run this after `beachball:bump`? Or a combination of the two.

Reply:
Done